Sources/Constants/OAColors.h

```diff
@@ -189,4 +189,7 @@
 
 #define plugin_icon_green 0x67AC5B
 
+#define color_ok 0x2BBE31
```

Comment:
If these colors are related to opening hours, you should name them accordingly (color_place_open, color_place_closed)